client-src/elements/form-field-specs.js

```diff
@@ -1128,6 +1128,50 @@ export const ALL_FIELDS = {
     >web_feature.mojom</a>.`,
   },
 
+  'ot_require_approvals': {
+    type: 'checkbox',
+    initial: false,
+    label: 'Trial participation requires approval',
+    help_text: html`
+      Will this trial require registrants to receive approval before
+      participating? See information about this setting at
+      <a target="_blank" href="">Some link</a>`,
```

Comment:
I don't think we want to provide a link here, but rather instruct feature authors to reach out to origin-trials-support@google.com to discuss their needs. Maybe replace "See information about this setting at Some link" with "Please reach out to origin-trials-support@google.com beforehand to discuss options here."

Reply:
Description updated to:

"Will this trial require registrants to receive approval before participating? Please reach out to origin-trials-support@google.com beforehand to discuss options here."